packages/coinstac-pipeline/src/controller.js

```diff
@@ -250,13 +263,15 @@ module.exports = {
           const errCb = (err) => {
             setStateProp('state', 'error');
             controllerState.activeComputations[controllerState.computationIndex].stop()
-              .then(() => rej(err));
+              .then(() => rej(err))
+              .catch(err => rej(err));
```

Comment:
The `err` variable is being overridden here. Just making sure you are aware.

Reply:
I am, but maybe a bad choice of var names